docs: jcloud docs revamp for instance

zac-li opened the pull request:
**Goals:**
JCloud docs update after introducing "instance".

Preview:
https://doc-jcloud-instance--jina-docs.netlify.app/concepts/jcloud/yaml-spec/

- [ ] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

fogx commented on docs/concepts/jcloud/yaml-spec.md:
```diff
@@ -34,32 +32,46 @@ Since each Executor has its own business logic, it may require different cloud r
+
+#### CPU Tiers
+
+| Instance | Cores | Memory | Credits Per Hour |
```
We introduce `Credits Per Hour` in this table, but do not explain where this is coming from or what they mean. I suggest we introduce the credits concept and/or link to the pricing page.

https://cloud.jina.ai/pricing

zac-li replied:
Yea that's definitely lacking, I will add this piece of info in!

alexcg1 replied:
I'll check this again after you add that info @zac-li

zac-li replied:
added some

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
```diff
@@ -3,21 +3,19 @@
 JCloud extends Jina's {ref}`Flow YAML specification<flow-yaml-spec>` by introducing the special field `jcloud`. This lets you define resources and scaling policies for each Executor and gateway.
 
-Here's a Flow with two Executors that have specific resource needs. `indexer` demands 10G `ebs` disk, whereas `encoder` demands two cores, 8G RAM and two dedicated GPUs. 
+Here's a Flow with two Executors that have specific resource needs. `indexer` demands 10G `ebs` disk, whereas `encoder` demands G5 instance, which implies 4 cores and 8G RAM are used. See below sections for further information about Instance Type.
```
```suggestion
Here's a Flow with two Executors that have specific resource needs: `indexer` requires a 10 GB `ebs` disk, whereas `encoder` requires a G5 instance, which implies that four cores and 8 GB RAM are used. See the below sections for further information about Instance Types.
```

alexcg1 replied:
Let's use [Wikipedia abbreviation conventions](https://en.wikipedia.org/wiki/Gigabyte) for consistency

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
```diff
@@ -34,32 +32,46 @@ Since each Executor has its own business logic, it may require different cloud r
+### Instance
 
-By default, `0.1 (1/10 of a core)` CPU is allocated to each Executor. You can use the `cpu` argument under `resources` to change it.
+JCloud uses the concept of "Instance" to respresent the hardware specifications. 
```
```suggestion
JCloud uses the concept of an "Instance" to represent a specific set of hardware specifications. 
```
Is this correct?

Also, please spell check next time :sweat_smile:

zac-li replied:
Yea that's correct, oops sorry I should have installed the spelling checker in the IDE.

zac-li replied:
fixed

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
```diff
@@ -34,32 +32,46 @@ Since each Executor has its own business logic, it may require different cloud r
 
-By default, `0.1 (1/10 of a core)` CPU is allocated to each Executor. You can use the `cpu` argument under `resources` to change it.
+JCloud uses the concept of "Instance" to respresent the hardware specifications. 
+In above example, C2 Instance Type stands for 4 cores and 8G RAM based on the CPU Tiers Instance defination table below.
```
```suggestion
In above example, a C2 Instance Type represents four cores and 8 GB RAM based on the CPU Tiers Instance definition table below.
```

Please:

- Spell check throughout
- Switch `8G` to `8 GB` (with space) throughout

zac-li replied:
cool fixed

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
````diff
@@ -69,83 +81,27 @@ If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G mem
 ```
 
-#### Shared GPU
+| Instance | GPU    | Memory | Credit Per Hour |
````
```suggestion
| Instance | GPU    | Memory | Credits per hour |
```

Sentence case for table headers throughout please

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
```diff
@@ -69,83 +81,27 @@ If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G mem
-#### Shared GPU
+| Instance | GPU    | Memory | Credit Per Hour |
+|----------|--------|--------|-----------------|
+| G1       | shared | 14G    | 100             |
+| G2       | 1      | 14G    | 125             |
+| G3       | 2      | 24G    | 250             |
+| G4       | 4      | 56G    | 500             |
```
Fix column 3 for `<number> GB`

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
````diff
@@ -69,83 +81,27 @@ If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G mem
-```
 
-#### Dedicated GPU
+To use `shared` GPU, `G1` needs to be specificied as Instance Type.
````
```suggestion
To use `shared` GPU, `G1` needs to be specificied as Instance Type.
```
Why are we capitalizing `Instance Type`?

zac-li replied:
I just thought it's a special term we come up with, or you think it should be`Instance type` or just all lower cased `instance type`?

alexcg1 replied:
Which of us came up with it? If no compelling reason otherwise, I'd go with `instance type` (all lower case)

zac-li replied:
I did, yea no compelling reason, switched to all lower case.

alexcg1 commented on docs/concepts/jcloud/yaml-spec.md:
````diff
@@ -69,83 +81,27 @@ If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G mem
-      resources:
-        memory: 8G
-```
+Using a dedicated GPU is the default way to provision GPU for an Executor. This automatically creates nodes or assigns the Executor to land on a GPU node. In this case, the Executor owns the whole GPU. 
````
```suggestion
Using a dedicated GPU is the default way to provision a GPU for an Executor. This automatically creates nodes or assigns the Executor to land on a GPU node. In this case, the Executor owns the whole GPU.
```

What do you mean by `land`?

zac-li replied:
I'm not exactly sure, let me check with the team.

zac-li replied:
there's nothing special to it, maybe we can just do:

```
assigns the Executor to a GPU node
```

?

alexcg1 replied:
Yup, that sounds better

zac-li replied:
fixed

fogx commented:
not sure if it belongs in this ticket, but can we remove the parent page `jina-ai-cloud/` and replace it with `/concepts/jcloud/`? the slug is bad and it adds no real information. 

I'm also not very happy with the title `YAML specification` on `concepts/jcloud/yaml-spec/` @alexcg1 . I get that this is what needs to be done, but it does not describe the task that a user will want to do -> configure their hardware in JCloud.

fogx commented on docs/concepts/jcloud/yaml-spec.md:
````diff
@@ -33,120 +31,94 @@ Since each Executor has its own business logic, it may require different cloud r
-Maximum of 16G RAM is allowed per Executor.
-```
+An Executor using a `shared` GPU shares this GPU with up to four other Executors.
+This enables time-slicing, which allows workloads that land on oversubscribed GPUs to interleave with one another.
````
```suggestion
This enables time-slicing, which allows workloads that land on oversubscribed GPUs to interleave with one another. 
To use `shared` GPU, `G1` needs to be specified as instance type.
```

-> move this sentence up

zac-li replied:
done

fogx commented on docs/concepts/jcloud/yaml-spec.md:
````diff
@@ -33,120 +31,94 @@ Since each Executor has its own business logic, it may require different cloud r
-```
+The tradeoffs with a `shared` GPU are increased latency, jitter, and potential out-of-memory (OOM) conditions when many different applications are time-slicing on the GPU. If your application is memory consuming, we suggest using a dedicated GPU.
+
+To use `shared` GPU, `G1` needs to be specified as instance type.
````
```suggestion
```
move sentence up. It is now between two blocks that talk about dedicated GPUs

zac-li replied:
done

fogx commented:
@alexcg1 i would also suggest that we restructure the hint about storage, so that we directly highlight the strengths and weaknesses of each storage type.
Something like this ->

Ephemeral (default)
| Pros  | Cons |
| ------------- | ------------- |
| Free of charge  | not persistent  |
| good IO performance  | not shared between Executors or Flows |
|  resizes dynamically |   |

efs 
| Pros  | Cons |
| ------------- | ------------- |
| shared between Executors and Flows | manual shrink/grow  |
| persistent | additional costs |
|  |  slowest IO performance  |

ebs 
| Pros  | Cons |
| ------------- | ------------- |
| best IO performance  | additional costs  |
| persistent |  not shared between Executors or Flows|
|   | manual shrink/grow  |


Do we have some benchmarks for these IO performances btw? @zac-li

zac-li commented:
@fogx @alexcg1 
Yea I don't think the title `YAML specification` fits well here. I think the structure was refactored when Han moved the jcloud docs around. I recommend re-constructing it, but not necessarily in this PR.

>Do we have some benchmarks for these IO performances
I don't think we have solid test data from our side, but we could link the stats analysis from Amazon maybe?
https://docs.aws.amazon.com/efs/latest/ug/performance.html
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSPerformance.html

alexcg1 commented:
> @alexcg1 i would also suggest that we restructure the hint about storage, so that we directly highlight the strengths and weaknesses of each storage type.

I like it!

fogx commented:
@alexcg1 can you finalize the title & the table?


@zac-li the links talk a lot about how to optimize the individual storage types, but not so much about the actual recommendations/values. If we do not give users the flexibility & visibility (and knowledge) on how to optimize storage in our ecosystem then this information will not be useful for them. -> efs can be max i/o or general purpose - how does this compare to ebs? Since we do not have anything right now then lets skip it and i will create a separate ticket for it.

zac-li commented:
> Do we have some benchmarks for these IO performances
Checked with the Team, we don't have any specific metrics from our side to share, so maybe we can just link AWS' pages?

So far we have a few major issues raised because of this PR:

1. File structure doesn't make sense
2. Storage section doesn't give clear guidance on why kind of storage users should pick
3. We haven't yet given users recommendation on instances users should pick ([ref](https://jinaai.slack.com/archives/C046ZPCJ951/p1678698913742619?thread_ts=1678430812.341659&cid=C046ZPCJ951))

I will be going on vacation starting from tomorrow. I'd say handle above 3 issues in different passes, especially for 3, we need to confirm with other Teams to in order to give a reasonable recommendation.

So except for the 3 issues, how are we looking here?

fogx commented:
one more minor issue:
I would like to add another Call-to-action in this section / at the end of JCloud to lead users to the pricing page or the landing page for our cloud hosting (which doesn't exist yet). @alexcg1 could you compose such a CTA? Just something like "Ready to get started, find our pricing here"? Or should users instead proceed to the YAML specification page? As we discussed, with the unfitting title i'm afraid very few would continue on to that page
https://doc-jcloud-instance--jina-docs.netlify.app/concepts/jcloud/#restrictions

alexcg1 commented:
Anything in particular jump out at you? I asked ChatGPT

Prompt: `Rewrite this to make it a more compelling call to action. Give me 10 variants: "Ready to get started with Jina AI Cloud? Check out our pricing here!"`

```
Take your first step towards revolutionizing your AI capabilities with Jina AI Cloud. Check out our pricing now!
Don't wait any longer to supercharge your AI development. Discover Jina AI Cloud's pricing today!
Are you ready to unlock the power of AI with Jina AI Cloud? Take a look at our pricing options now!
Ready to elevate your AI game? Explore our pricing plans for Jina AI Cloud today!
It's time to experience the future of AI development. Check out Jina AI Cloud's pricing options now!
Want to stay ahead of the competition? Start using Jina AI Cloud today by checking out our pricing!
Achieve greater AI performance and efficiency with Jina AI Cloud. Browse our pricing options now!
Ready to transform the way you build and deploy AI? Discover Jina AI Cloud's pricing plans now!
The power of AI is at your fingertips. Get started with Jina AI Cloud by exploring our pricing!
Are you ready to take your AI projects to the next level? Learn about our pricing options for Jina AI Cloud today!
```

alexcg1 commented:
Looks good!

zac-li commented:
> Are you ready to unlock the power of AI with Jina AI Cloud? Take a look at our pricing options now!

Is this a generic question that applies to all the offerings on Jina AI Cloud, or just to JCloud?
If it's the former, the call to action should probably happen at https://doc-jcloud-instance--jina-docs.netlify.app/jina-ai-cloud/

fogx commented:
lgtm @zac-li 

>Is this a generic question that applies to all the offerings on Jina AI Cloud, or just to JCloud?

the point is that we need to catch users at any point where they might be ready to transition to JAC/JCloud. Users might not read the JAC page, but they still want to deploy on JCloud (the pricing is the same, the two are very much connected). But i agree that it should also be a CTA (the same one is fine) on `/jina-ai-cloud`

Sorry for the feature creep, but since we're talking about it:
I would like to remove this banner. We released JAC now and there will always be changes.
![image](https://user-images.githubusercontent.com/10303647/226867206-1016acfc-0333-4cef-8505-026b10b66c83.png)

And i would like to remove "free" from this.
> JCloud is a cost-efficient hosting platform specifically designed for Jina projects.
![image](https://user-images.githubusercontent.com/10303647/226867430-b4779804-0773-4dc4-b50e-59af5ff4b969.png)

zac-li commented:
@fogx ok fixed 👍🏼

fogx commented on docs/jina-ai-cloud/index.md:
```diff
@@ -20,7 +20,7 @@ Executor Hub is an Executor marketplace that allows you to share, explore and te
 :link-type: doc
 
-JCloud is a free CPU/GPU hosting platform for Jina projects.
+JCloud is a CPU/GPU hosting platform for Jina projects.
```
```suggestion
JCloud is a cost-efficient hosting platform specifically designed for Jina projects.
```

fogx replied:
sorry should have done a suggestion right from the start. After this change it can be merged from my side

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -459,11 +422,11 @@ executors:
 ```
 
-You can pass the `enable: true` argument to `gateway`, so as to only enable the tracing support in gateway:
+You can pass the `enable: true` argument to `gateway`, so as to only enable the tracing support in the Gateway:
````
```suggestion
You can pass the `enable: true` argument to `gateway` to only enable tracing support in the Gateway:
```

alexcg1 commented on docs/concepts/jcloud/index.md:
```diff
@@ -309,16 +309,16 @@ jc scale good-martin-ca6bfdef84 --executor executor0 --replicas 2
 ## Restrictions
 
-JCloud scales according to your needs. You can demand different resources (GPU/RAM/CPU/storage/instance-capacity) based on the needs of your Flows and Executors. If you have specific resource requirements, please contact us [on Slack](https://jina.ai/slack) or raise a [GitHub issue](https://github.com/jina-ai/jcloud/issues/new/choose).
+Jina AI Cloud scales according to your needs. You can demand different instance type with GPU/Memory/CPU pre-defined based on the needs of your Flows and Executors. If you have specific resource requirements, please contact us [on Slack](https://jina.ai/slack) or raise a [GitHub issue](https://github.com/jina-ai/jcloud/issues/new/choose).
```
```suggestion
Jina AI Cloud scales according to your needs. You can demand different instance types with GPU/memory/CPU predefined based on the needs of your Flows and Executors. If you have specific resource requirements, please contact us [on Slack](https://jina.ai/slack) or raise a [GitHub issue](https://github.com/jina-ai/jcloud/issues/new/choose).
```

zac-li commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -33,120 +31,94 @@ Since each Executor has its own business logic, it may require different cloud r
+    memory: 8G
 ```
 
+We would translate the raw numbers from input to Instance tier that fits most. 
````
> Why is Instance capitalized? Think I asked this before

Yea it's discussed before, ok let me fix it

zac-li replied:
fixed.

zac-li commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -246,10 +206,21 @@ Below are the defaults and requirements for the configurations:
 After JCloud deployment using the autoscaling configuration, the Flow serving part is just the same: the only difference you may notice is it takes a few extra seconds to handle the initial requests since it needs to scale the deployments behind the scenes. Let JCloud handle the scaling from now on, and you can deal with the code!
 
+### Pricing
+At this moment, pricing for autoscaled Executor/Gateway follows the same {ref}`JCloud pricing rules <jcloud-pricing>` for the most part.
+We track the minimum number of replicas in Autoscale configurations and use it as multiplier for the replicas used when calculating the
+`Credits Per Hour`.
```
fixed

zac-li commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -33,120 +31,94 @@ Since each Executor has its own business logic, it may require different cloud r
-      resources:
-        gpu: 1
-```
+#### GPU Tiers
````
fixed

alexcg1 commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -161,37 +133,25 @@ If your Executor needs to share data with other Executors and retain data persis
+#### Pricing
+Here are the numbers in terms of per GB Credits Per Month used for three kinds of storage described above.
+
+| Instance  | Credits Per Month Per GB|
```
Huggingface are using sentence case. Only first letter capitalized, not first letter of every word

zac-li replied:
ok let me fix that

zac-li replied:
fixed

zac-li commented:
@alexcg1 another look?

alexcg1 commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -33,119 +31,93 @@ Since each Executor has its own business logic, it may require different cloud r
 
-### CPU
+JCloud uses the concept of an "instance" to represent a specific set of hardware specifications.
+In above example, a C4 instance type represents two cores and 4 GB RAM based on the CPU Tiers instance definition table below.
```
```suggestion
In the above example, a C4 instance type represents two cores and 4 GB RAM based on the CPU tiers instance definition table below.
```

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -33,119 +31,93 @@ Since each Executor has its own business logic, it may require different cloud r
-```{hint}
-Maximum of 16 cores is allowed per Executor.
-```
+Note that if you are still using the legacy resource specification interface, such as:
````
```suggestion
Note that we will translate the raw numbers from input to instance tier that fits most closely if you are still using the legacy resource specification interface, such as:
```

Suggest wrapping this in an admonition. I'd add the code but since it includes lots of backticks GH won't render it. Just `grep admonition` in the `docs/` folder and you'll see some examples

zac-li replied:
ok fixed

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -33,119 +31,93 @@ Since each Executor has its own business logic, it may require different cloud r
+    memory: 8G
 ```
 
+We would translate the raw numbers from input to instance tier that fits most.
````
```suggestion
```

I moved this up to connect it better

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -33,119 +31,93 @@ Since each Executor has its own business logic, it may require different cloud r
 ```
 
+We would translate the raw numbers from input to instance tier that fits most.
+There are circumstances the instance tier does not exactly fulfill the CPU cores and Memory you need, like in above example.
+But we would "ceil" the requests to the lowest tier that would satisfy all the specifications. 
+In this case, `C6` would be considered, as `C5`'s `Cores` is lower than what's being requested (4 vs 8).
````
```suggestion
There are circumstances in the instance tier where they don't exactly fulfill the CPU cores and memory you need, like in the above example.
In cases like this we "ceil" the requests to the lowest tier that satisfies all the specifications. 
In this case, `C6` would be considered, as `C5`'s `Cores` are lower than what's being requested (4 vs 8).
```

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -161,37 +133,25 @@ If your Executor needs to share data with other Executors and retain data persis
-          kind: efs
-```
+#### Pricing
+Here are the numbers in terms of per GB Credits Per Month used for the three kinds of storage described above.
````
```suggestion
Here are the numbers in terms of credits per GB per month for the three kinds of storage described above.
```

It is credits per GB per month, right?

zac-li replied:
yup, thanks for catching this

alexcg1 commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -246,10 +206,21 @@ Below are the defaults and requirements for the configurations:
 | max    | 2           | int, up to 5             | Maximum number of replicas                        |
 | metric | concurrency | `concurrency`  /   `rps` | Metric for scaling                                |
-| target | 100         | int                      | Target number after which replicas autoscale      |
+| target | 100         | int                      | Target number each replicas tries to maintain     |
```
```suggestion
| target | 100         | int                      | Target number of replicas to try to maintain     |
```

alexcg1 commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -246,10 +206,21 @@ Below are the defaults and requirements for the configurations:
-| target | 100         | int                      | Target number after which replicas autoscale      |
+| target | 100         | int                      | Target number each replicas tries to maintain     |
 
 After JCloud deployment using the autoscaling configuration, the Flow serving part is just the same: the only difference you may notice is it takes a few extra seconds to handle the initial requests since it needs to scale the deployments behind the scenes. Let JCloud handle the scaling from now on, and you can deal with the code!
```
```suggestion
After a JCloud deployment using the autoscaling configuration, the Flow serving part is just the same: the only difference you may notice is it takes a few extra seconds to handle the initial requests since it needs to scale the deployments behind the scenes. Let JCloud handle the scaling from now on, and you can deal with the code!
```

alexcg1 commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -246,10 +206,21 @@ Below are the defaults and requirements for the configurations:
 After JCloud deployment using the autoscaling configuration, the Flow serving part is just the same: the only difference you may notice is it takes a few extra seconds to handle the initial requests since it needs to scale the deployments behind the scenes. Let JCloud handle the scaling from now on, and you can deal with the code!
 
+### Pricing
+At this moment, pricing for autoscaled Executor/Gateway follows the same {ref}`JCloud pricing rules <jcloud-pricing>` for the most part.
+We track the minimum number of replicas in autoscale configurations and use it as multiplier for the replicas used when calculating the
```
```suggestion
At present, pricing for autoscaled Executor/Gateway follows the same {ref}`JCloud pricing rules <jcloud-pricing>` for the most part.
We track the minimum number of replicas in autoscale configurations and use it as a multiplier for the replicas used when calculating the
```

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -246,10 +206,21 @@ Below are the defaults and requirements for the configurations:
+### Restrictions
+```{admonition} **Restrictions**
+
+- Autoscale does not currently allow the use of `ebs` as a storage kind in combination. Please use `efs` and `ephemeral` instead.
````
```suggestion
- Autoscale does not currently allow the use of `ebs` as a storage type in combination. Please use `efs` and `ephemeral` instead.
```

alexcg1 commented on docs/concepts/jcloud/configurations.md:
````diff
@@ -246,10 +206,21 @@ Below are the defaults and requirements for the configurations:
+```{admonition} **Restrictions**
+
+- Autoscale does not currently allow the use of `ebs` as a storage kind in combination. Please use `efs` and `ephemeral` instead.
+- If the Gateway uses a multi-protocol setup, autoscale is not supported.
````
```suggestion
- Autoscale is not supported for multi-protocol Gateways.
```

alexcg1 commented on docs/concepts/jcloud/index.md:
`````diff
@@ -85,12 +85,12 @@ jina flow --uses flow.yml
 
 ````{tip}
-The best practice of creating a JCloud project is to use:
+The best practice of creating a Jina AI Cloud project is to use:
`````
```suggestion
The best practice for creating a Jina AI Cloud project is to use:
```

alexcg1 commented on docs/concepts/jcloud/index.md:
````diff
@@ -85,12 +85,12 @@ jina flow --uses flow.yml
 jc new
 ```
-This ensures the correct project structure accepted by JCloud.
+This ensures the correct project structure accepted by Jina AI Cloud.
````
```suggestion
This ensures the correct project structure that is accepted by Jina AI Cloud.
```

alexcg1 reviewed (changes requested):
A few more fixes I missed the first time

zac-li commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -161,37 +133,25 @@ If your Executor needs to share data with other Executors and retain data persis
+#### Pricing
+Here are the numbers in terms of per GB Credits Per Month used for the three kinds of storage described above.
+
+| Instance  | Credits per month per GB|
```
changed to per GB per month

zac-li commented on docs/concepts/jcloud/configurations.md:
```diff
@@ -161,37 +133,25 @@ If your Executor needs to share data with other Executors and retain data persis
 - The disk cannot be shared with other Executors or Flows.
-- You must pass a storage size parameter (default: `1G`, max `10G`).
-
+- Default storage size is `5G`.
```
changed to 5 GB

zac-li commented:
@alexcg1 addressed the comments, please re-review.

I accidentally push other commits in from other folks (even though I rebased in the end so overall diff is correct) which caused commit lint to fail. Shall I open another PR for the exact changes and link this over?

alexcg1 reviewed (changes requested):
One more tiny thing

zac-li commented on docs/concepts/jcloud/index.md:
```diff
@@ -309,16 +309,16 @@ jc scale good-martin-ca6bfdef84 --executor executor0 --replicas 2
 
+## Configurations
+
+Please refer to {ref}`Configurations <jcloud-configurations>` for configuring the Flow on Jina AI Cloud.
```
fixed!

alexcg1 reviewed (approved):
LGTM :+1:
